lightning/checkpoints/checkpoints.go

```diff
@@ -182,6 +217,40 @@ func (cpd *TableCheckpointDiff) String() string {
 	)
 }
 
+// Apply the diff to this checkpoint.
+func (cp *TableCheckpoint) Apply(cpd *TableCheckpointDiff) {
```

Comment:
only apply to already exist `engine checkpoint`  and  `chunk checkpoint`, what about those not exist checkpoint?

Reply:
or any comments for this function

Reply:
emphasized it only affects existing checkpoints in the comment.